backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -453,6 +453,158 @@ describe("Listings", () => {
     })
   })
 
+  // REMOVE_WHEN_EXTERNAL_NOT_NEEDED
```

Comment:
The reason for the copypasta is that the expectation (at least for MVP) is that any behavior we expect from `/listings` is also expected from `/listings/combined`.  There are only a couple of extra tests that need to be run on top of that, namely verification that `isExternal` is false on any new listings we add and that the number of listings from `/listings/combined` is 1) larger than the number of listings from `/listings` and 2) all of the extra listings have `isExternal` set to `true`.  That last test is pending some in progress work on the seeder.

Reply:
Re: query builder unit tests, I'm going to disagree with that.  There are no unit tests anywhere else in the code for query builders or anything else like that, and the results are all verified in the e2e tests anyway since they rely on the results from `CombinedListingQueryBuilder`.  The JSON objects are also not unmarshaled by `CombinedListingQueryBuilder`, so that would require tests on TypeORM libraries that aren't owned by this application.